### ISSUE TITLE
Transient CI error on Debian Bullseye

### ISSUE THREAD

**Issue by apyrgio** [freedomofpress/dangerzone#388]
Our CI tests on Debian Bullseye sometimes fail with:

```
Failed to detect the owner for the current cgroup: stat /sys/fs/cgroup/systemd/system.slice/cloud-final.service: no such file or director
```

Re-running the CI tests sometimes works, but we need to figure out why this happens.

**Comment by apyrgio** [comment]
Turns out that this is a warning. The actual error(s) are:

```
unable to write volume event: \"write unixgram @0009a->/run/systemd/journal/socket: sendmsg: no such file or directory\
```

A Podman issue that matches the Podman version in Debian Bullseye (3.0.1) and the error message seem to be this: https://github.com/containers/podman/issues/16304. From that issue, we see that running Podman 3.0.1 in more restrictive environments can lead to such errors. The proposed solution is to update Podman through another repo, but we can't make that assumptions for our users.

**Comment by apyrgio** [comment]
I looked more into this error, and I have the following update:

1. The trigger for the error is indeed `cloud-final.service`. It seems that the `/sys/fs/cgroup` directory, which is present in the host, is mounted in the container as well, and it's part of the way Podman uses cgroups. The `cloud-final.service` slice is owned by `root`, but within the Podman container it's owned by `nobody`. My understanding is that this file suddenly disappears and re-appears, and Podman 3.0.1 does not handle it properly.
2. CircleCI has more [machine images ](https://circleci.com/developer/images?imageType=machine) than Ubuntu 20.04. I've tested out Ubuntu 22.04, and I didn't encounter the above issue. 
3. While testing out Podman on Ubuntu 22.04, I've stumbled on the following new issues:
   * Podman uses Systemd as cgroup manager: https://github.com/containers/podman/issues/6368
   * Podman sometimes cannot get the exit code of a process: https://github.com/containers/podman/issues/10987
   
   The first error affects Podman both when running in the host and in the Dangerzone environment. The second _seems_ to affect Podman only when running in the Dangerzone environment. In order to not overcomplicate things, we can use the following Podman configuration in all of the environments:
   
   ```ini
   [engine]
   cgroup_manager="cgroupfs"
   events_logger="file"
   ```

This way (using Ubuntu 22.04 with the above Podman config), we can reliably run Dangerzone tests on Debian Bullseye.
